### ISSUE TITLE
[FxA-Settings] The page header is at plural

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/fxa#7676]
**Build**
 Stage [Train 1.201.0]

**Affected platforms**
 Windows 10 64-bit
 macOS 10.14.6

**Steps to reproduce**

1. Navigate to [https://accounts.stage.mozaws.net/beta/settings/](https://accounts.stage.mozaws.net/beta/settings/) and login using an existing account.
1. Check the page header.

**Expected results**
 The page header is at singular - 'Firefox Account' - see [figma](https://www.figma.com/file/Nj2WjMOXoJBiPoWvyJ946X/FxA---Manage-Settings-(Copy)?node-id=0%3A1)

**Actual Results**
 The page header is at plural - 'Firefox Accounts'.

┆Issue is synchronized with this [Jira Bug](https://jira.mozilla.com/browse/FXA-3107)
┆Attachments: <a href="https://jira.mozilla.com/secure/attachment/15369/Screenshot_16.png">Screenshot_16.png</a> | <a href="https://jira.mozilla.com/secure/attachment/16240/Screenshot_9.png">Screenshot_9.png</a>
┆Issue Number: FXA-3107

**Comment by Jane-Kotovich** [comment]
Heya @jodyheavener @LZoog ! Could I work on this issue please! Im an Outreachy applicant and would love to grab this one as my second issue! Thanks, Jane

**Comment by jodyheavener** [comment]
@Jane-Kotovich all yours!

**Comment by Jane-Kotovich** [comment]
Thanks @jodyheavener ! A quick q - if I changed tsx file, do I have to run some yarn install to see changes? Simple save and yarn start doesn't show me result even though I know its been made up. Thanks!

**Comment by LZoog** [comment]
@Jane-Kotovich you shouldn't have to `yarn install` after any changes, you should just be able to hit "refresh" and see them! Are you looking at the Settings page (`/settings`, not `/beta/settings`)? This is only currently used there. :)

BTW, you'll also want to update the localization file, you can find it searching for the localized ID.

**Comment by Jane-Kotovich** [comment]
@LZoog Thanks heaps for advice! Will check it!

**Comment by Jane-Kotovich** [comment]
@LZoog @jodyheavener  hey, looks like Im having some issues here .. I tried bunch of thing but none of this changes anything..
<img width="1440" alt="Screen Shot 2021-04-07 at 12 19 39 pm" src="https://user-images.githubusercontent.com/76935202/113802040-58c69100-979d-11eb-9494-af0df3d1207e.png">
<img width="1440" alt="Screen Shot 2021-04-07 at 12 19 51 pm" src="https://user-images.githubusercontent.com/76935202/113802074-6714ad00-979d-11eb-8319-97d8438e08b7.png">

I even tried to change all Accounts to Account but it didn't make any results. 
Only one way I can see differences is when I change it in google chrome dev tools, which obviously not the point. :(

**Comment by Jane-Kotovich** [comment]
@LZoog @jodyheavener  hey friends, I spent all day searching for localisation file and localised id "logo-text". Change most of them in whole project to Firefox Account but haven't had any success ;(
At the end I have tried as well .title & headerTitle and different varieties of all that.
Any tips would be extremely appreciated! Thanks

**Comment by jodyheavener** [comment]
@Jane-Kotovich so it looks like the change for this is fairly simple. We just need to change the `header-title` ID to point to the singular `product-firefox-account` ID [here](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/components/HeaderLockup/en-US.ftl#L7:L7).

However because you are not in the US you may not see the change take effect due to the locale headers your browser sends; for myself (as I am not in the US either) I had to actually go and make this same change in `fxa-settings/public/locales/en-GB/settings.ftl` to be able to preview it.

**Comment by Jane-Kotovich** [comment]
@jodyheavener  yeah I definitely tried it there and it didn't work - now it explains everything. Thanks will do now!

**Comment by LZoog** [comment]
Here's the screenshot: 
![image](https://user-images.githubusercontent.com/13018240/118514462-2e4fe800-b6fa-11eb-9922-e4d6be140532.png)

It looks like we need both the header and page title updated. Maybe we need to update another string somewhere else for the header?

**Comment by Ayushsunny** [comment]
> Here's the screenshot:
> ![image](https://user-images.githubusercontent.com/13018240/118514462-2e4fe800-b6fa-11eb-9922-e4d6be140532.png)
> 
> It looks like we need both the header and page title updated. Maybe we need to update another string somewhere else for the header?

Hello @LZoog what about this [PR](https://github.com/mozilla/fxa/pull/8184) ?
